akka-docs/rst/java/persistence.rst

```diff
@@ -918,6 +922,37 @@ directory. This can be changed by configuration where the specified path can be
 Note that it is not mandatory to specify a snapshot store plugin. If you don't use snapshots
 you don't have to configure it.
 
+.. _journal-proxy:
```

Comment:
I think those must be unique, so include the `java` as we have done in other places